js/search/search.js

```diff
@@ -268,7 +268,7 @@ class Search {
 
             if (!fs.existsSync(this.indexFolderName) || !fs.existsSync(this.realTimeIndex)) {
                 log.send(logLevels.ERROR, 'Index folder does not exist.');
-                reject('Index folder does not exist.');
+                reject(new Error('Index folder does not exist.'));
```

Comment:
When we are throwing these errors, are we in any case indicating that the process failed? I think it would be good to do that, if at all it is discernible to the user that something like a "real time indexing" is happening.

Reply:
@VikasShashidhar We have a ticket for displaying indexing process and errors to the users. Will be handling this there 

Ticket Number : [SEARCH-551](https://perzoinc.atlassian.net/browse/SEARCH-551)

Reply:
👍